appConfig/appConfig.tst1.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
We don't need this since POCs will be deployed to SND environments only

Reply:
@kbabigumira removed